services/py-api/src/database/repository/teams_repository.py

```diff
@@ -54,7 +60,7 @@ async def update(
         obj_id: str,
         input_data: BaseModel,
         session: Optional[AsyncIOMotorClientSession] = None,
-        **kwargs: Dict[str, Any]
+        **kwargs: Dict[str, Any],
```

Comment:
why we have a comma here?

Reply:
maybe the  pre commit hook added it. I dont remember adding a comma. But I will change it

Reply:
![image](https://github.com/user-attachments/assets/2f8182f9-e18c-4990-a70c-7462356aa8d8)
Yes boss its the pre-commit hook who gets angry